storage/redis.go

```diff
@@ -59,52 +59,133 @@ func (r *redisStore) CheckRtt(ctx context.Context) int64 {
 
 var sidekiqHeartbeatJobsKey = ":work"
 
-func (r *redisStore) SendHeartbeat(ctx context.Context, heartbeat *Heartbeat) error {
+func (r *redisStore) SendHeartbeat(ctx context.Context, heartbeat *Heartbeat, heartbeatManagerTTL time.Duration) error {
+	now, err := r.GetTime(ctx)
+	if err != nil {
+		return err
+	}
 
 	pipe := r.client.Pipeline()
 	rtt := r.CheckRtt(ctx)
 
-	managerIdentity := r.namespace + heartbeat.Identity
+	managerKey := r.getManagerKey(heartbeat.Identity)
 	sidekiqProcessesKey := r.namespace + "processes"
-
 	pipe.SAdd(ctx, sidekiqProcessesKey, heartbeat.Identity) // add to the sidekiq processes set without the namespace
 
-	pipe.HSet(ctx, managerIdentity, "beat", heartbeat.Beat.UTC().Unix())
-	pipe.HSet(ctx, managerIdentity, "quiet", heartbeat.Quiet)
-	pipe.HSet(ctx, managerIdentity, "busy", heartbeat.Busy)
-	pipe.HSet(ctx, managerIdentity, "rtt_us", rtt)
-	pipe.HSet(ctx, managerIdentity, "rss", heartbeat.RSS)
-	pipe.HSet(ctx, managerIdentity, "info", heartbeat.Info)
-	pipe.Expire(ctx, managerIdentity, 60*time.Second) // set the TTL of the heartbeat to 60
+	pipe.HMSet(ctx, managerKey, "beat", heartbeat.Beat.UTC().Unix())
```

Comment:
Any reason to switch the first command to `HMSet`?

Reply:
I actually intended to change all the HSets to a single HMSet, but was probably side-tracked mid-change. 😆 
Implemented as HMSet in commit 8ae85e1c6e9047312b29e23e4256a882d7272e08.

Reply:
@keitabyte this removed some of the functionality where the heartbeat would automatically expire itself. Was that intentional or accidental? It broke compatibility with the Sidekiq Web UI